### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by caseyturnerarrington**
Added an item about disabling pop-up blocker

Closes: #???

## Description

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: docs/content/flow-port/faq.md [hunk 1]
```diff
@@ -149,3 +149,7 @@ Please see a list [here](https://github.com/onflow/flow/blob/master/nodeoperator
        - a.) No you do not! You only need your Ledger device to sign transactions. If you simply want to view your account, you can do so without your Ledger. You can do this by navigating directly to the appropriate desired page URL, while inputting your address into the URL itself. For easy reference, below is a list of these URLs and where you would input your address:
        - Dashboard: https://port.onflow.org/account/[AccountAddress]
        - Stake & Delegate: https://port.onflow.org/stake-delegate/[AccountAddress]
+
+  7. I am clicking 'submit' to execute a transaction, but nothing is happening. How can I unblock myself?
+  
+       - a.) Please disable any pop-up blockers you have and refresh the page. If you are still experiencing issues, please reach out via our [discord](https://discord.gg/4yGnMzkZxr) in the appropriate channel.
```
Comment (JeffreyDoyle) [inline docs/content/flow-port/faq.md:155]: can we add to disable any ad blockers as well?

**Review by JeffreyDoyle — APPROVED** [review]

File: docs/content/flow-port/faq.md (hunk 1 above)
Comment (psiemens) [inline docs/content/flow-port/faq.md:155]:
```suggestion
       - a.) Please disable any pop-up blockers you have and refresh the page. If you are still experiencing issues, please reach out via [Discord](https://discord.gg/4yGnMzkZxr) in the appropriate channel.
```

Comment (psiemens) [reply docs/content/flow-port/faq.md:155]: Maybe we should add a Port channel to Discord too?

Comment (caseyturnerarrington) [reply docs/content/flow-port/faq.md:155]: I already brought that up to wei but he said no for now.

**Review by JeffreyDoyle — APPROVED** [review]

**Pull request merged**